test/MLOps.NET.SQLite.IntegrationTests/SQLiteMetaDataStoreTests.cs

```diff
@@ -136,6 +136,23 @@ public async Task CreateRunAsync_WithoutGitCommitHash_ShouldProvideEmptyGitCommi
             run.GitCommitHash.Should().Be(string.Empty);
         }
 
+        [TestMethod]
+        public async Task CreateExperimentAsync_Twice_ShouldNotAddDuplicate()
+        {
+            // Arrange
+            var sut = new MLOpsBuilder()
+	            .UseSQLite()
+	            .UseModelRepository(new Mock<IModelRepository>().Object)
+	            .Build();
+
+            //Act
+            var experimentId = await sut.LifeCycle.CreateExperimentAsync("test");
+	        var experimentId2 = await sut.LifeCycle.CreateExperimentAsync("test");
```

Comment:
Just a minor comment here about the indentation being one or two spaces too many

Reply:
Should be fixed now